### DISCUSSION TITLE
All Day event creation

### DISCUSSION THREAD

**Discussion by tjzetty** [kuzmoyev/google-calendar-simple-api#101] [Q&A] [answered]
I am trying to make a program that automatically creates events. Sometimes these events do not have a time so I want to mark them as 'All Day' like I usually would in google calendar. Is there an attribute I am missing? How can I create events without a start or end that are 'All Day'? I added an image to show what I am wanting to create.
![image](https://user-images.githubusercontent.com/65916505/119709313-0fa0ce00-be2b-11eb-850c-735d43f8901e.png)

**Comment by kuzmoyev** [comment, marked as answer]
What do you mean without `start` or `end`? When is the event happening then?

To create an `all-day` event just pass the `date` object to `start`:


```
e = Event(
    'My all day event",
    start=date(2021, 8, 31)
)
```

Please, let me know if that works.

**Reply by tjzetty** [reply]
That is exactly what I wanted. Thanks!
